float64 flightP
float64 flightD
float64 stanceP
float64 stanceD
float64 toeFilterGain
float64 toeThreshold
float64 legLen
float64 force
float64 legP
float64 legD

# 0 for left, 1 for right
int8    activeLeg
